client/src/components/ClubPage.tsx

```diff
@@ -141,33 +182,63 @@ export default function ClubPage() {
           isOwner={memberType === "owner"}
         />
       )}
+      {showCreateAnnModal && (
```

Comment:
I'd just rename this (and other similar ones) `showCreateAnnouncementModal `, it's not egregiously long and it'd match the rest of the variables